src/test/java/posters/pageobjects/pages/checkout/NewBillingAddressPage.java

```diff
@@ -40,9 +40,10 @@
 
     @Override
     @Step("ensure this is a new billing address page")
-    public void isExpectedPage()
+    public NewBillingAddressPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.